src/data.js

```diff
@@ -0,0 +1,31 @@
+const data = {
```

Comment:
- learned how we can use data.js type separate files for keeping data abstract
- great use of modular and cleaner approach
- minimal and good use of CSS variables